src/AppInstallerCLICore/Commands/UpgradeCommand.cpp

```diff
@@ -20,7 +20,8 @@ namespace AppInstaller::CLI
         bool ShouldListUpgrade(Context& context)
         {
             return context.Args.Empty() ||
-                (context.Args.GetArgsCount() == 1 && context.Args.Contains(Execution::Args::Type::Source));
+                (context.Args.GetArgsCount() == 1 && (context.Args.Contains(Execution::Args::Type::Source) || context.Args.Contains(Execution::Args::Type::IncludeUnknown))) ||

```

Comment:
My gut on the right way to do this is to enable:
```
for (Execution::Args::Type type : context.Args.GetTypes())
{
    if (type != Execution::Args::Type::Source && type != Execution::Args::Type::IncludeUnknown)
    {
        return false;
    }
}
return true;
```
I definitely don't like keeping the count in the condition and referencing types multiple times.

Reply:
Resolved in latest commit.